app/services/activity_service.rb

```diff
@@ -0,0 +1,34 @@
+class ActivityService
```

Comment:
Todo: Test coverage for this (which can emulate the `UserAnalyticsPresenter` to start)